### DISCUSSION TITLE
Change uploader name

### DISCUSSION THREAD

**Discussion by ajsteele** [ente-io/ente#585] [Enhancements]
I have had several friends enter the wrong name when uploading via the web interface, and I don’t think there’s currently a way to change this as the album owner?

Also, it might be worth reconsidering the text that prompts the user for this?
![image](https://github.com/ente-io/photos-app/assets/6997662/b3f48ab9-7185-4c08-a7ea-d16bcbe37058)

I’ve had a couple of friends enter the name of the place or event, so perhaps they only read ‘enter name’ and not the grey text underneath! Could it be replaced with something as simple as ‘what is your name?’ or ‘enter your name’, and put the grey text under the input field?

**Comment by JacquesLucke** [comment]
I agree that the dialog is a bit ambiguous at a first glance currently.

Changing the placeholder seems to be a simple and maybe already effective enough solution:
![image](https://github.com/user-attachments/assets/a1642c9a-bb81-40d2-b672-a5c30aaf69d4)

I tested that in #6208.

**Comment by JacquesLucke** [comment]
Related to that, I also think that being able to change the uploader name after the fact (at least for the owner) is important.

I had some initial code for this in #6203.

The main tricky thing is to make this functionality available in the UI without affecting the existing UI too much, as this is probably something that one does quite rarely. Something like this could work:

https://github.com/user-attachments/assets/1c2fbcd5-5174-4011-9745-8c36d777ea0c

The other tricky thing is that it seems most useful to be able to rename the uploader of all files with the same uploader name at once, while currently the full name is only displayed when viewing the data for a file.

**Reply by vishnukvmd** [reply]
Hey, thanks for this, we'd want to tweak the copies a bit, but this broadly is how we'd want to see this feature. We'll see how we can squeeze this into our roadmap :)
